go/libraries/doltcore/sqle/logictest/main/main.go

```diff
@@ -91,6 +91,7 @@ func NewDoltRecordResult(e *logictest.ResultLogEntry) *DoltResultRecord {
 		TestFile:     e.TestFile,
 		LineNum:      e.LineNum,
 		Query:        e.Query,
+		Duration: 	  e.Duration.String(),
```

Comment:
Would make this a numeric type and specify units. We don't have a duration type in dolt.